appium/tests/specs/mock/setup/CheckRevokedKeyUpdate.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { ekmKeySamples } from 'api-mocks/apis/ekm/ekm-endpoints';
+import { MockApi } from 'api-mocks/mock';
+import {
+  SplashScreen
+} from '../../../screenobjects/all-screens';
+import MailFolderScreen from "../../../screenobjects/mail-folder.screen";
+import NewMessageScreen from "../../../screenobjects/new-message.screen";
+import SetupKeyScreen from "../../../screenobjects/setup-key.screen";
+
+describe('SETUP: ', () => {
+
+  it('check revoked key update logic', async () => {
```

Comment:
Let's rename it as `will not update a revoked public key with valid one`

Reply:
Ok